app/src/main/java/org/groundplatform/android/persistence/remote/firebase/schema/SurveyConverter.kt

```diff
@@ -68,5 +71,6 @@ internal object SurveyConverter {
       jobMap = jobMap.toPersistentMap(),
       acl = surveyProto.aclMap.entries.associate { it.key to it.value.toString() },
       dataSharingTerms = dataSharingTerms,
+      dataVisibility = dataVisibility,
```

Comment:
Can we mark the model field as non-null?

Reply:
As we are not saving this field in local db, hence it is null.

Reply:
If we are not saving it, then how will we get the value in offline state?

Reply:
Maybe for offline, the code is using the default value which uses `show` behavior?

Reply:
Thanks for the extra context. I'd like to spend some more time reviewing it to ensure there is no risk.

Reply:
Let's store this field in the local db.